app/components/preview-pictures.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component'
+import { computed } from '@ember/object'
+import { inject as service } from '@ember/service'
+
+export default Component.extend({
+  store: service(),
+  allPhotos: computed('album.id', function() {
+    return this.get('store').query('photo', {
+      albumId: this.get('album.id')
+    })
+  }),
+  preview: computed.filter('allPhotos', function(photo, index) {
+    return index < 4
+  })
```

Comment:
This should not be necessary.. Did you try it with `.slice(0, 4)`?

Reply:
Yes I tried it. Did not work.